Selenium_pract_task/Tets_Cases/Validate_global_search_works_as_expected.cs

```diff
@@ -1,34 +1,23 @@
 ﻿using NUnit.Framework;
 using Selenium_pract_task.Entities.AbstractEntities;
-using Selenium_pract_task.POM.EpamMainPage.EpamCareersPage;
 using Selenium_pract_task.POM.EpamMainPage;
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
 
 namespace Selenium_pract_task.Tets_Cases
 {
     public class Validate_global_search_works_as_expected : BaseTest
     {
-        [Test]
-        public void Test1()
+        [TestCase("BLOCKCHAIN")]
+        [TestCase("Cloud")]
+        [TestCase("Automation")]
+        public void TestGlobalSearchFunctionality(string searchFieldValue)
         {
-            //Arrange
-            string programinLanguage;
-
-            //Act
-            programinLanguage = "Java";
-
-            //Assert
             driver.Navigate().GoToUrl("https://www.epam.com/");
             var epamMainPageContext = new EpamMainPageContext(driver);
-            epamMainPageContext.verifyCookiesHendler()
-                .clickOnIconMagnifier()
-                .setTextInputSearchField("Automation")
-                .clickButtonFind()
-                .verifyAllSerchedLinksContainsValue("Automation");
+            epamMainPageContext.VerifyCookiesHandler()
+                .ClickOnIconMagnifier()
+                .SetTextInputSearchField(searchFieldValue)
+                .ClickButtonFind()
+                .VerifyAllSerchedLinksContainsValue(searchFieldValue);
```

Comment:
VerifyAllSerchedLinks**ContainsValue** contains which value? Name it like this: containsSearchText or something like that